receiver/hostmetricsreceiver/internal/scraper/filesystemscraper/filesystem_scraper_test.go

```diff
@@ -27,6 +27,7 @@ import (
 )
 
 func TestScrape(t *testing.T) {
+	t.Skip("")
```

Comment:
Why is this skipped?

Reply:
Thanks for catching this! I had to skip this test while running the tests locally and forgot to remove this. I've removed it now.